client/app.jsx

```diff
@@ -21,12 +22,14 @@ class App extends React.Component {
 
   componentDidMount() {
 
+    console.log(this.state.listingId)
+
     var listingId = this.state.listingId
 
     // console.log('axios called for listingId: ', listingId)
-    axios.get(`http://localhost:8888/api/reservation/${listingId}`)
+    axios.get(`/api/reservation/${listingId}`)
       .then(res => {
```

Comment:
I see you changed from a URL to a relative path, but I'm curious if there was a reason why you put the full URL to start with? It's something i noticed in a previous review forgot to ask then.

Reply:
I actually can't remember why, but realized it was causing me to change the port in two places so I altered it today.  Totally call those things out and I'll switch em.